src/lib.rs

```diff
@@ -161,8 +161,10 @@ impl State {
     }
 
     async fn init_friendships(&mut self) {
+        // clear friendships, start again.
+        self.friendships.clear();
```

Comment:
Why do we need to do this?

Reply:
Since we have a list of available friendships it is almost free to start fresh and to not concentrate all friendships on the first created users.